docker-compose.yaml

```diff
@@ -13,7 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-version: '3.2'
```

Comment:
The removal of the top-level `version` seems to be breaking the docker compose.

Reply:
That's odd. I get an warning when I run with the version.
```
pmoore@PhilsStkMacbook minder_test % make run-docker
Running docker compose down
WARN[0000] /Users/pmoore/Source/minder_test/docker-compose.yaml: `version` is obsolete 
```
And the [spec](https://github.com/compose-spec/compose-spec/blob/main/spec.md#version-and-name-top-level-elements) shows it as unnecessary. 

What command are you running and what's the error message you are getting?

Reply:
This seems like it's a directive to `docker-compose`, probably to turn on/off functionality.

Reply:
I believe the issue was that I put invalid data in the server-config.yaml.example file, and it was failing a type check. I've updated the file and the docs to reflect the new values. That should resolve it hopefully.